### PR TITLE
Added example config management use case, links, and example install instructions

### LINKED ISSUE #623: Add example usecase for configuration management
Example installation order via `GitRepo` CRs: 
1. Calico CRDs / controllers installed
2. Setting global network policy at a cluster level (need Calico CRDs installed first)
3. GateKeeper installed (huge emphasis on cluster labels and **Overlays** as a big Fleet features --> "which clusters get each part of each bundle")
4. Ingress setup and system daemons setup

In the above example, Fleet is used for the configuration management behind clusters. Network policies, traffic rules, system configuration, etc. is handled by Fleet. We should include an example workflow either inspired by or closely following the one above.

### PR DESCRIPTION
**Pull request by jtravee**
Closes #623 

Added to the examples with https://github.com/rancher/fleet-examples/blob/master/multi-cluster/helm/fleet.yaml use case. Also added example install directions for `GitRepo` CRs.

References:
https://fleet.rancher.io/examples/#deploy-kubernetes-manifests-across-clusters-with-customization
https://fleet.rancher.io/concepts/

### CODE REVIEW

File: docs/concepts.md [hunk 1]
```diff
@@ -2,20 +2,30 @@
 
 Fleet is fundamentally a set of Kubernetes custom resource definitions (CRDs) and controllers
 to manage GitOps for a single Kubernetes cluster or a large scale deployment of Kubernetes clusters
-(up to one million). Below are some of the concepts of Fleet that will be useful through out this documentation.
+(up to one million).
```
Comment (nickgerace) [inline docs/concepts.md:5]: We should likely change this to "tested up to one million clusters", but better wording than what I just wrote 😅. I think we should make it clear that Fleet's design is _technically_ mass scalable (horizontal), but we have _tested_ that amount.

File: docs/concepts.md [hunk 2]
```diff
@@ -2,20 +2,30 @@
...
+    the Fleet manager and Fleet controllers are used fairly interchangeably.
 * **Single Cluster Style**: This is a style of installing Fleet in which the manager and downstream cluster are the
     same cluster.  This is a very simple pattern to quickly get up and running with GitOps.
 * **Multi Cluster Style**: This is a style of running Fleet in which you have a central manager that manages a large
     number of downstream clusters.
 * **Fleet agent**: Every managed downstream cluster will run an agent that communicates back to the Fleet manager.
     This agent is just another set of Kubernetes controllers running in the downstream cluster.
 * **GitRepo**: Git repositories that are watched by Fleet are represented by the type `GitRepo`.
+
+>**Example installation order via `GitRepo` custom resources when using Fleet for the configuration management behind clusters:**
```
Comment (nickgerace) [inline docs/concepts.md:22]:
```suggestion
>**Example installation order via `GitRepo` custom resources when using Fleet for configuration management of downstream clusters:**
```

File: docs/concepts.md [hunk 3]
```diff
@@ -2,20 +2,30 @@
...
+>
+> 1. Install Calico CRDs and controllers.
```
Comment (nickgerace) [inline docs/concepts.md:24]: We should link Calico: https://github.com/projectcalico/calico

File: docs/concepts.md [hunk 4]
```diff
@@ -2,20 +2,30 @@
...
+> 2. Set the global network policy at a cluster level.
```
Comment (nickgerace) [inline docs/concepts.md:25]:
```suggestion
> 2. Set one or multiple cluster-level global network policies
```

File: docs/concepts.md [hunk 5]
```diff
@@ -2,20 +2,30 @@
...
+> 3. Install GateKeeper. Note that **cluster labels** and **overlays** are critical features in Fleet as they determine which clusters will get each part of the bundle.
```
Comment (nickgerace) [inline docs/concepts.md:26]: We should link Gatekeeper: https://github.com/open-policy-agent/gatekeeper

File: docs/concepts.md [hunk 6]
```diff
@@ -2,20 +2,30 @@
...
+> 4. Set up and configure the ingress and system daemons.
```
Comment (nickgerace) [inline docs/concepts.md:27]:
```suggestion
> 4. Set up and configure ingress and system daemons.
```

**Review by nickgerace — CHANGES_REQUESTED** [review]

File: docs/concepts.md (hunk 1 above)
Comment (jtravee) [reply docs/concepts.md]: Revised per feedback.

File: docs/concepts.md (hunk 2 above)
Comment (jtravee) [reply docs/concepts.md]: Revised per feedback.

File: docs/concepts.md (hunk 4 above)
Comment (jtravee) [reply docs/concepts.md]: Revised as suggested.

File: docs/concepts.md (hunk 6 above)
Comment (jtravee) [reply docs/concepts.md]: Revised as suggested.

File: docs/concepts.md (hunk 3 above)
Comment (jtravee) [reply docs/concepts.md]: Linked as suggested.

File: docs/concepts.md (hunk 5 above)
Comment (jtravee) [reply docs/concepts.md]: Linked as suggested.

**Review by nickgerace — APPROVED** [review]
LGTM! We should get a review from @StrongMonkey as well

**Review by StrongMonkey — APPROVED** [review]
LGTM

**Pull request merged**